lib/pf/lookup/person.pm

```diff
@@ -35,7 +35,8 @@ Lookup informations on a person
 =cut
 
 sub lookup_person {
-    my ($pid, $source_id) = @_;
+    my ($pid, $source_id, $context) = @_;
+    use Data::Dumper; get_logger->info(Dumper(\@_));
```

Comment:
debug maybe

Reply:
Removed it instead, it was just for my own debug